analyzer/src/main/kotlin/managers/DotNet.kt

```diff
@@ -62,7 +62,14 @@ class DotNet(
     private val reader = DotNetPackageFileReader()
 
     override fun resolveDependencies(definitionFile: File, labels: Map<String, String>): List<ProjectAnalyzerResult> =
-        listOf(resolveNuGetDependencies(definitionFile, reader, NuGetSupport.create(definitionFile)))
+        listOf(
```

Comment:
Should we add a hint to the commit message that this is useful to match the behavior of tools like https://help.sonatype.com/iqserver/analysis/nuget-application-analysis#NuGetApplicationAnalysis-Evaluation:Projectfiles for easier comparison of results?